docs/Developer Guide.md

```diff
@@ -5,9 +5,9 @@ have its own additional setup and tooling.
 
 ## Setup
 
-Finch always uses the latest Node.js LTS version. Issues running against older
-Node.js versions are immediately closed. Issues running against newer Node.js
-versions are addressed as time permits.
+Finch always supports >= the oldest Node.js LTS version. Issues running against
+older Node.js versions are immediately closed. Issues running against newer
+Node.js versions are addressed as time permits.
```

Comment:
Will fix. 10.15 was the oldest at the time of writing. I updated to 12.15 when opening the PR.